pysgrid/sgrid.py

```diff
@@ -7,14 +7,16 @@
 from __future__ import (absolute_import, division, print_function)
 
 from netCDF4 import Dataset
+import numpy as np
 
 from .read_netcdf import NetCDFDataset, parse_padding, find_grid_topology_var
-from .utils import calculate_angle_from_true_east, pair_arrays
+from .utils import calculate_angle_from_true_east, pair_arrays, points_in_polys
 from .variables import SGridVariable
 
 
 class SGrid(object):
 
+    grid_names = ['node', 'face', 'edge1', 'edge2']
```

Comment:
> . I see these as grid components of 1 grid. Not sure if calling it grid names is the way to go. Might be confusing for people that known the SGRID conventions to read the code.

I also think of those as components of one grid and would be confusing for people familiar with the current SGRID conventions. Perhaps `grid_component` would be a more apt name?